internal/querycoord/index_info.go

```diff
@@ -0,0 +1,13 @@
+package querycoord
+
+import (
+	"github.com/milvus-io/milvus/internal/proto/commonpb"
+)
+
+type extraIndexInfo struct {
+	indexID        UniqueID
+	indexName      string
+	indexParams    []*commonpb.KeyValuePair
+	indexSize      uint64
+	indexFilePaths []string
+}
```

Comment:
it is no need add a new file index_info.go, you can just create extraIndexInfo struct in file index_check.go, and the new field has not liscense